src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2366,6 +2371,55 @@ public override BoundNode VisitConditionalOperator(BoundConditionalOperator node
             return null;
         }
 
+        private BoundNode VisitRefConditionalOperator(BoundConditionalOperator node)
```

Comment:
I have tried that initially and combined method had too much conditional logic. It seems easier to have just two methods side by side.

---
In reply to: [105269425](https://github.com/dotnet/roslyn/pull/17436#discussion_r105269425) [](ancestors = 105269425)